app/views/users/show.html.erb

```diff
@@ -8,4 +8,15 @@
       </h1>
     </section>
   </aside>
+
+  <div class="col-md-8">
+    <% if @microposts.any? %>
+      <h3><%= t ".microposts" %> (<%= @microposts.count %>)</h3>
```

Comment:
phân biệt count, size, length

Reply:
Khi sử dụng count. Nó sẽ dùng câu lệnh sql (SELECT COUNT() FROM…) để truy vấn ra số lượng của activerecord mà chúng ta truy vấn. Một điều quan trọng nữa là count không được lưu trữ trong đối tượng mà chúng ta gọi, Nghĩa là một lần gọi count thì nó sẽ thực hiện một truy vấn (SELECT COUNT() FROM…) vào database dù cho trước đó bạn đã gọi hay chưa.
khi sử dụng length, nó sẽ đếm số lượng mà không sử dụng thêm câu truy vấn nếu đối tượng đó đã được nạp, ngược lại, nó sẽ nạp toàn bộ object của bạn vào bộ nhớ và thực hiện đếm. Khi gọi lenght lần thứ 2 trở đi, lenght sẽ không còn load từ database nữa, mà trả về luôn kết quả